### PR TITLE
Fix broken command in documentation

### LINKED ISSUE #1100: Bad command in documentation
In documentation in start instructions [Option 2: start Selenoid container](https://aerokube.com/selenoid/latest/#_option_2_start_selenoid_container):
you have the command: 
```bash
docker run -d                                   \
--name selenoid                                 \
-p 4444:4444                                    \
-v /var/run/docker.sock:/var/run/docker.sock    \
-v ./config/:/etc/selenoid/:ro              \
aerokube/selenoid:latest-release
```

It produces the error:
```
docker: Error response from daemon: create ./config/: "./config/" includes invalid characters for a local volume name, only "[a-zA-Z0-9][a-zA-Z0-9_.-]" are allowed. If you intended to pass a host directory, use absolute path.
```

Broken commands in documentation is not a good idea 😄

### PR DESCRIPTION
**Pull request by p4m-dev**
Fixes #1100

### CODE REVIEW

File: docs/starting-selenoid-manually.adoc [hunk 1]
```diff
@@ -107,7 +107,7 @@ docker run -d                                   \
 --name selenoid                                 \
 -p 4444:4444                                    \
 -v /var/run/docker.sock:/var/run/docker.sock    \
--v ./config/:/etc/selenoid/:ro              \
+-v $(pwd)/config/:/etc/selenoid/:ro              \
```
Comment (BorisOsipov) [inline docs/starting-selenoid-manually.adoc:110]: @p4m-dev `$(pwd)` was removed intentionally in https://github.com/aerokube/selenoid/commit/69405aefa6caab1adcce0cc48d971aa0ba492692. I guess it should be `/your/directory/` for consistency

**Review by BorisOsipov — COMMENTED** [review]
minor change request

File: docs/starting-selenoid-manually.adoc (hunk 1 above)
Comment (p4m-dev) [reply docs/starting-selenoid-manually.adoc:110]: Yeah, it makes sense

**Comment by p4m-dev** [comment]
I'm not sure but looks like PowerShell command needs the same changes

**Comment by p4m-dev** [comment]
I think this PR can be merged. If there is the problem with PowerShell command. I can fix it in another issue/PR

**Comment by p4m-dev** [comment]
I have not device with Windows installed to test works PowerShell command or no...

**Comment by vania-pooh** [comment]
@p4m-dev thank you for contribution.

**Pull request merged**